composer.json

```diff
@@ -12,7 +12,7 @@
     "spryker/gui": "^3.0.0",
     "spryker/kernel": "^3.33.0",
     "spryker/money": "^2.0.0",
-    "spryker/payment": "^4.0.0",
+    "spryker/payment": "^5.0.0",
```

Comment:
This is a BC break, the original idea was to only allow the new major version.

Reply:
This triggers a (cascading) major here, yeah.